src/app.js

```diff
@@ -27,6 +30,15 @@ app.set("view engine", "ejs");
 
 app.use(cookieParser());
 
+const getProjectIDs = withCache(getData.getProjectIDs, {
+  cacheStorage: projectIDsCache,
+  ttl: 1800,
+});
+const getJobs = withCache(getData.getJobs, {
+  cacheStorage: jobsCache,
+  ttl: 5,
```

Comment:
Now this is completely detached from the default auto reload interval. They should share the same config, so that developers would not miss when updating the value.

BTW, ideally both of the cache ttl should come from server config.

Reply:
@CMDRyan This one is not resolved yet btw.